draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1678,8 +1678,8 @@ Content-Type: text/plain
    they only appear within safe field value contexts (e.g., data not required by HTTP).
 </t>
 <t>
-   Leading and trailing whitespace in raw field values is removed upon field
-   parsing (e.g., <xref target="field.parsing"/>). Field definitions where leading or trailing
+   Leading and trailing whitespace in field values &MUST; be removed before the
+   field value is used (but not forwarded). Field definitions where such
    whitespace in values is significant will have to use a container syntax such
    as quoted-string (<xref target="quoted.strings"/>).
```

Comment:
This language to me might suggest that forwarders should not trim whitespace and that it may have some semantic value. It seems better to explicitly call out that scenario as a MAY exception.

Something like:

```suggestion
   Leading and trailing whitespace in field values &MUST; be removed before the
   field value is used. Field definitions where such
   whitespace in values is significant will have to use a container syntax such
   as quoted-string (<xref target="quoted.strings"/>).
   
   A request being forwarded &MAY; leave whitespace intact.
```